### ISSUE TITLE
[BUG] Inconsistent rendering of error messages in `daml test` and LSP scenario service

### ISSUE THREAD

**Issue by lucianojoublanc-da** [digital-asset/daml#13208]
I confirm that, if this is a bug that has security implications, I already contacted security@digitalasset.com and followed the [responsible disclosure policy](https://digitalasset.com/security/).

I confirm that this is not a question or a request for technical support by the community, for which the [Daml forum](https://discuss.daml.com/) is available.

# Affected Daml version
2.0

# Bug description

Failing an assertion in a `Script` using `===` will print out a message using the `Show` instance in `daml test`, but some other function in the LSP server (tested using `daml ide` + vim). See https://github.com/digital-asset/contingent-claims/issues/46#issuecomment-1061922660 for sample output.

# To reproduce
- Write a custom `Show` instance (not sure if this is required)
- Write a failing assertion (tested using `===`)
- Check error message in IDE scenario service
- Check error message from `daml test`

breaking `(_ : Claim) ===` asserts in https://github.com/digital-asset/contingent-claims/blob/better_show/test/daml/Test/Lifecycle.daml should give you an example of this.

# Expected behavior
We'd expect both cases to use the custom `Show` instance. It can be really difficult to debug large recursive structures when it includes things like hashes and field names.

**Comment by cocreature** [comment]
I cannot reproduce this on your project. I am not entirely convinced symlinking works reliably here so I suspect what might have happened is that the changes just didn’t get propagated in the IDE. Can you reproduce after restarting vscode?

**Comment by lucianojoublanc-da** [comment]
You're right - restarting the LSP server seems to have fixed this. How annoying :disappointed: . Sorry about the noise.
